### PR TITLE
[CDAP-1161] remove MetricsScope from MetricsCollectorService interface

### PR DESCRIPTION
**Pull request by abaranau**
This is the first step of moving scope to metric name. PR contains mechanical change (this change is done with "refactor..." of IDE by removing metrics scope from MetricsCollectorService (https://github.com/caskdata/cdap/compare/develop...feature/mv-metric-scope#diff-13)). Wanted to isolate this change from other - to help review.

Will not be merged until this one is finished: https://github.com/caskdata/cdap/pull/991

The idea is to move metrics scope (user/system) to a metric prefix. At emit time - it is a tag for user metrics, and absent for all other cases. At processing time it is moved to metric name prefix.

### CODE REVIEW

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsCollectionService.java [hunk 1]
```diff
@@ -110,8 +110,8 @@ protected Scheduler scheduler() {
   }
 
   @Override
-  public final MetricsCollector getCollector(final MetricsScope scope, final Map<String, String> tags) {
-    return collectors.getUnchecked(new CollectorKey(scope, tags));
+  public final MetricsCollector getCollector(final Map<String, String> tags) {
+    return collectors.getUnchecked(new CollectorKey(MetricsScope.SYSTEM, tags));
```
Comment (gokulavasan) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsCollectionService.java]: So we still have MetricsScope.SYSTEM here?

**Comment by gokulavasan** [comment]
One comment. Other changes look good.

File: cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsCollectionService.java (hunk 1 above)
Comment (abaranau) [inline cdap-watchdog/src/main/java/co/cask/cdap/metrics/collect/AggregatedMetricsCollectionService.java]: yup - that is bridging changes of this step to this one: https://github.com/caskdata/cdap/pull/991

I.e. fixed in 2nd step

**Comment by abaranau** [comment]
merged in https://github.com/caskdata/cdap/pull/991 -reviewed also. Will run build one more time and merge as both were reviewed

**Comment by abaranau** [comment]
only couple tests failisng  - same as in develop (https://builds.cask.co/download/CDAP-DUT496-JOB1/build_logs/CDAP-DUT496-JOB1-5.log) - have to merge now

**Pull request merged**